modules/offline/post/index.js

```diff
@@ -49,6 +50,10 @@ const postOfflineForms = async () => {
     const uploadedForms = await uploadOfflineForms(offlineForms).catch(() => ({
       status: "Error",
     }));
+    getAWSLogger().log({
+      type: "OFFLINE_FORM_UPLOADED",
+      parseUser: user.objectId
+    });
```

Comment:
@nisargap how does this detect latency? we don't have the start and end time for the request here